public/data/vasantisuthar.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "VASANTI SUTHAR",
+  "type": "personal",
+  "bio": "Open Source Enthusiast!",
+  "avatar": "https://github.com/vasantisuthar.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/vasantisuthar",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/vasanti_suthar",
+      "icon": "twitter"
+    },
+    {
+      "name": "Follow me on Showwcase",
+      "url": "https://www.showwcase.com/vasanti-suthar",
+      "icon": "showwcase"
+    },
+    {
+      "name": "Follow me on Linkedin",
```

Comment:
It'd look better if we capitalize `i` here 😅 What do you think?
```suggestion
      "name": "Follow me on LinkedIn",
```

Reply:
yeah right